group_vars/all/rpc-o.yml

```diff
@@ -30,10 +30,21 @@ horizon_custom_uploads:
 repo_build_pip_extra_indexes:
   - "https://rpc-repo.rackspace.com/pools"
 
-# Tempest test execution options
-tempest_run_tempest_opts:
-  - "--serial"
-# Tempest testr options
-tempest_testr_opts: []
+# Elasticsearch versions
```

Comment:
Putting these variables *only* in the role caused the repo process to fail when indexing the roles for `*_apt_pacakges` variables and their corresponding packages [here](https://github.com/rcbops/rpc-openstack/blob/master/rpcd/playbooks/roles/elasticsearch/vars/ubuntu-16.04.yml#L17) The repo process tried to interpolate these variables when there were being indexed, but failed since they were defined outside the scope of the repo roles. I had to place them her to compromise.